qa/integration-tests/src/test/java/io/zeebe/broker/it/ClientRule.java

```diff
@@ -90,6 +94,36 @@ public void waitUntilTopicsExists(final String... topicNames) {
         .until(t -> t != null && t.keySet().containsAll(expectedTopicNames));
   }
 
+  public void waitUntilDeploymentIsDone(long key) {
```

Comment:
Can this be the default behavior to avoid that I need to call it in every test?

Reply:
In most of the cases (in broker test suite) I tried to do so, but the problem is that  for example in the integration test suite the ZeebClient it self is used, so how I could intercept here?

Reply:
True :)